### ISSUE TITLE
New mockup for keyboard help dialog

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/resistance-in-a-wire#104]
We have settled on a style for the keyboard help dialog, @arouinfar and @terracoda can you please create a new mockup for the "concise" version of the dialog in this sim?

**Comment by arouinfar** [comment]
Here's a mockup using the Ohm's Law controls, with Home and End split into two lines. @terracoda can you please review?
![screen shot 2017-10-03 at 5 19 07 pm](https://user-images.githubusercontent.com/8419308/31153556-0cf21168-a85f-11e7-9d22-3027332dbdf5.png)

**Comment by terracoda** [comment]
@arouinfar, another suggestion from the design crit was to use consistent language, so use "adjust" or "move", but not both. We could use, "move" or "adjust", or "adjust value of", "adjust values in", and "adjust value to" for the Slider Controls. I picked "move" as it seemed like the simplest language.

**Slider Controls**
- Move sliders: 
- Move in smaller steps: 
- Move in larger steps:
- Move to minimum value:
- Move to maximum value:

**General Navigation**
- Move to next item:
- Move to previous item:
- Exit this dialog:

And just making a stab at what the text might be for the PDOM version
**Slider Controls**
- Move sliders with the left, up, down, or right arrow key
- Move in smaller steps with shift plus an arrow key
- Move in larger steps with Page Up or Page Down key
- Move to minimum value with Home key
- Move to maximum value with End key

**General Navigation**
- Move to next item with Tab key
- Move to previous item with Shift plus Tab key
- Exit this dialog with Escape key

Not sure we need "key" in all cases.

**Comment by terracoda** [comment]
@arouinfar and a little more line spacing would be good, too.

**Comment by arouinfar** [comment]
Thanks for the feedback @terracoda! I think the updated text is a a lot cleaner and simpler. I'll change the wording and add a bit more padding to the mockups.

**Comment by arouinfar** [comment]
@terracoda back to you for review!
![screen shot 2017-10-04 at 10 54 40 am](https://user-images.githubusercontent.com/8419308/31188535-756dccd0-a8f2-11e7-80f0-2ec8137d8eaf.png)

**Comment by terracoda** [comment]
@arouinfar, looks super nice!

**Comment by terracoda** [comment]
@jessegreenberg, I think it is ready for implementation.

**Comment by jessegreenberg** [comment]
Alright, done in the above commits,  @arouinfar @terracoda could you please review? Any final pixel polishing?
![capture](https://user-images.githubusercontent.com/6396244/31301906-cd433a70-aaba-11e7-9c3e-823c1b88ef7a.PNG)

Can be seen in this version:  http://www.colorado.edu/physics/phet/dev/html/resistance-in-a-wire/1.3.0-dev.15/resistance-in-a-wire_en.html

**Comment by arouinfar** [comment]
Looks good to me @jessegreenberg!

**Comment by jessegreenberg** [comment]
Excellent thanks!  Closing issue.

**Comment by terracoda** [comment]
At todays's keyboard navigation we made some changes to the keyboard dialog representation of how to use two-sliders sliders. This change will affect other keyboard dialogs with two-way sliders e.g. Ohm's Law, Molecules and Light (to add github issues)

Changes are summarized as:
- Pair the Arrow keys as 2 options: [Left Right] or [Up Down]
- Make a two-line stack with one label for the Move in smaller steps
- Visually make the arrows more pointy, i.e., more Isosceles triangle-like
- Note: No changes to representations of 4-way interactions

So the slider section will be like this:

**Slider Controls**
Move sliders:  [Left Right] or [Up Down]
Move in smaller steps: Shift + [Left Right] or Shift + [Up Down] (stack these 2 optiopns)

**Comment by terracoda** [comment]
@arouinfar re-opening and assigning to you. I will update the previous comment with the affected issues.

**Comment by arouinfar** [comment]
Thanks for summarizing @terracoda! I've made the requested changes. I think the left/right arrow keys have a bit more contrast with one another, so I decided to put those first. Can you review and assign others from the meeting who may also be interested in reviewing?
![screen shot 2017-10-10 at 11 34 12 am](https://user-images.githubusercontent.com/8419308/31401091-f6262c4e-adae-11e7-9438-b37c70a304bb.png)

Here's a comparison of the original arrows (top) and pointier arrows (bottom).
![screen shot 2017-10-10 at 11 19 46 am](https://user-images.githubusercontent.com/8419308/31401110-0d76098c-adaf-11e7-8b1c-b8349d1af99b.png)

**Comment by jessegreenberg** [comment]
> Pair the Arrow keys as 2 options: [Left Right] or [Up Down]

Why are they paired this way?  Shouldn't it be [Left Down] and [Up Right] to indicate decrement/increment respectively?

**Comment by jessegreenberg** [comment]
I could be wrong, maybe I didnt' understand the reasoning for breaking them up in the first place.

**Comment by arouinfar** [comment]
Since there's an "or" between the options, I don't think you want to pair the redundant controls. Using left/right, you can traverse the entire slider track. Using left/up, you'd only be able to decrease the slider value.

**Comment by jessegreenberg** [comment]
Ok thanks @arouinfar.  I was mistaken, I thought the goal was to break it into "Left and down decrease" while "up and right increase".  But pairing them as two options for navigating the slider range makes sense.

**Comment by terracoda** [comment]
@arouinfar, looks awesome!

**Comment by terracoda** [comment]
Ooops, did you intentionally remove the "s" on "Move sliders"?

**Comment by terracoda** [comment]
@emily-phet, @ariel-phet, @amanda-phet, @oliver-phet, @zepumph comments welcome on the new mock-up in https://github.com/phetsims/resistance-in-a-wire/issues/104#issuecomment-335551751

Please comment and/or un-assign yourself.

**Comment by terracoda** [comment]
@arouinfar, I think the new Arrow keys and the order look great. Just wondering about the missing "s".

**Comment by oliver-phet** [comment]
I like this and do think it is clearer than all together.

**Comment by amanda-phet** [comment]
I like the new triangles! It all looks great to me. 

I'm noticing that the "move in larger steps" row uses the word "or", and am wondering if "and" would be more appropriate here since we use "or" to indicate alternative controls, not complementary controls. Or perhaps no word at all between the buttons (we don't use words between the arrow buttons after all).

**Comment by terracoda** [comment]
@amanda-phet, hmm, I think of them as two alternatives. I think we say "or" for other things like "Space" or "Enter" for buttons in other cases.

**Comment by amanda-phet** [comment]
Maybe I don't know what those buttons do.. I assumed page up was a large jump up, and page down is a large jump down. We don't use "or" between a regular up and down arrow.

**Comment by oliver-phet** [comment]
I agree. Leaving out "or" between page up/down would be the most consistent.

**Comment by terracoda** [comment]
@amanda-phet, Oh I agree with you. I think we need an "and" for [Pg Up] and [Pg Down]. They are complimentary.

**Comment by terracoda** [comment]
Or leaving it blank like @oliver-phet said. Both work for me.

**Comment by jessegreenberg** [comment]
Here is the new dialog based on the updated mockup and above comments, interested parties have thoughts or request additional pixel polishing?

![capture](https://user-images.githubusercontent.com/6396244/31473271-a75314da-aeb0-11e7-90b2-4aa288fe87a0.PNG)

Version to see it live: http://www.colorado.edu/physics/phet/dev/html/resistance-in-a-wire/1.3.0-dev.16/resistance-in-a-wire_en.html

**Comment by terracoda** [comment]
Looks great, @jessegreenberg.

I think the ESC item, should read "Exit this dialog: [Esc icon]".

Other than that, looks great to me.

**Comment by arouinfar** [comment]
Looks great @jessegreenberg!

@terracoda I'm not sure if we need to use "this" for the ESC item. Sims have multiple dialogs -- Keyboard Shortcuts, About, and sometimes there are additional dialogs in the sim (e.g. Show Light Spectrum in MAL). All dialogs can be exited with ESC. I doubt that "this" would cause confusion, though -- users would likely try ESC if they close other dialogs.

**Comment by terracoda** [comment]
@arouinfar, our original text was something like:

> [Esc] key to exit a dialog, like this one.

We have iterated and simplified a lot since then, so sure let's see how the more general "Exit a dialog: [Esc icon]" works. I am fine with that approach.

@jessegreenberg, looks good to me!

**Comment by ariel-phet** [comment]
Looks good to me!

**Comment by jessegreenberg** [comment]
Thanks everyone, sounds like this is ready to go. Closing.

**Comment by arouinfar** [comment]
@jessegreenberg in today's keyboard navigation meeting we further refined the language used for slider controls. This will apply to all sims with sliders that look like sliders, including Ohm's Law and MAL. 

![screen shot 2017-10-17 at 11 23 47 am](https://user-images.githubusercontent.com/8419308/31679224-b2fdaecc-b32d-11e7-9583-7a9a6a1b99ba.png)

**Comment by zepumph** [comment]
I'm on it

**Comment by zepumph** [comment]
Here is master:
![image](https://user-images.githubusercontent.com/6856943/31694771-9e924180-b352-11e7-8601-e177bf080aa0.png)


I'll leave open to make sure that this gets cherryPicked into the rc branch. Back to you @jessegreenberg.

**Comment by arouinfar** [comment]
Looks great @zepumph!

**Comment by jessegreenberg** [comment]
Thanks @zepumph, looks great! We will cherry pick into 1.3 and 1.3-phetio when ready.

**Comment by terracoda** [comment]
Thanks @zepumph, @jessegreenberg and @arouinfar! I agree. Looks great.

**Comment by jessegreenberg** [comment]
Closing, we are in RC phase!

**Comment by jessegreenberg** [comment]
Wait, this needed to be cherry picked??

**Comment by jessegreenberg** [comment]
Yes it did, marking as fixed-pending-testing.

**Comment by jessegreenberg** [comment]
Changes in both release branches (1.3-phetio and 1.4)
